### PR TITLE
API for Security/Team Interaction

### PR DESCRIPTION
**Pull request by chayim**
Given our most recent conversation on the API for interacting with the security team I've documented my current thoughts.

### CODE REVIEW

File: handbook/engineering/security/index.md [hunk 1]
```diff
@@ -89,11 +89,31 @@ On the security team, we work by planning, tracking, and reviewing - creating a
...
+
+The security team helps other teams investigate, orient, review, and sometimes plan changes. 
+
+**Small changes**
+
+Small changes, defined as less than 1 day of effort are communicated in a GitHub issue, either in the [sourcegraph repository](https://www.github.com/sourcegraph/sourcegraph) or, when sensitive in nature, the [security repository](https://www.github.com/sourcegraph/security-issues).  We provide a detailed bug report, including the steps to reproduce, and an example of the desired state.  Where possible, we identify and propose a fix, and submit a pull request to the [code owner](../code_reviews.md#who-should-i-get-a-code-review-from).
+
+**Medium changes**
+
+Medium changes operate on the same principle as small changes, with the exception of the team not submitting code changes. These changes also include a documented test plan for validation of the security change.
```
Comment (tsenart) [inline handbook/engineering/security/index.md:104]:
```suggestion
Medium changes operate on the same principle as small changes, except that the code owner is always the one driving it. These changes also include a documented test plan for validation of the security change. 
```

File: handbook/engineering/security/index.md [hunk 2]
```diff
@@ -89,11 +89,31 @@ On the security team, we work by planning, tracking, and reviewing - creating a
...
+
+**Medium changes**
+
+Medium changes operate on the same principle as small changes, with the exception of the team not submitting code changes. These changes also include a documented test plan for validation of the security change. 
+
+**Large changes**
+
+Large changes are communicated through [RFCs](../../communication/rfcs/index.md).  The RFC provide as much context as possible about the issue, the reason for the change, and its relative importance and urgency.  We collaborate on the RFC as a team, sharing the RFC with the Engineering Manager(s) for the impacted team for both feedback and [prioritizing](../../product/prioritizing.md).  Upon approval, we create GitHub issues, labeling them with a new label, for the RFC (i.e rfc214), sharing with the Engineering manager.  When possible, a member of the security team attends the planning session, to help provide input should more be needed.
```
Comment (tsenart) [inline handbook/engineering/security/index.md:108]:
```suggestion
Large changes are communicated through [RFCs](../../communication/rfcs/index.md).  The RFC provides as much context as possible about the issue, the reason for the change, and its relative importance and urgency.  We collaborate on the RFC as a team, sharing the RFC with the Engineering Manager(s) for the impacted team for both feedback and [prioritizing](../../product/prioritizing.md).  Upon approval, we create GitHub issues, labeling them with a new label, for the RFC (i.e RFC-214), sharing with the Engineering Manager.  When possible, a member of the security team attends the planning session, to help provide input should more be needed.
```

File: handbook/engineering/security/index.md [hunk 3]
```diff
@@ -89,11 +89,31 @@ On the security team, we work by planning, tracking, and reviewing - creating a
...
+
+**Providing input**
+
+When input is needed from the team on a change the team is added to a pull request.  We also respond to RFCs when assigned.
```
Comment (tsenart) [inline handbook/engineering/security/index.md:112]:
```suggestion
Teams planning and executing changes should ask for Pull Request reviews from the Security Team as needed to gain confidence in the implementation. Teams should also feel free to ask for a security review of any RFC they're working on that aren't directly created from the Security Goals.
```

File: handbook/engineering/security/index.md [hunk 4]
```diff
@@ -89,11 +89,31 @@ On the security team, we work by planning, tracking, and reviewing - creating a
...
 
 ## Members
 
-- [Chayim Kirshen](../../../company/team/index.md#chayim-kirshen-he-him) ([Engineering Manager](../roles.md#engineering-manager)).
+- [Shim Kirshen](../../../company/team/index.md#chayim-kirshen-he-him) ([Engineering Manager](../roles.md#engineering-manager)).
```
Comment (tsenart) [inline handbook/engineering/security/index.md:116]: Did you change your name?

**Review by tsenart — APPROVED** [review]

File: handbook/engineering/security/index.md [hunk 5]
```diff
@@ -89,11 +89,31 @@ On the security team, we work by planning, tracking, and reviewing - creating a
...
+
+**Small changes**
+
+Small changes, defined as less than 1 day of effort are communicated in a GitHub issue, either in the [sourcegraph repository](https://www.github.com/sourcegraph/sourcegraph) or, when sensitive in nature, the [security repository](https://www.github.com/sourcegraph/security-issues).  We provide a detailed bug report, including the steps to reproduce, and an example of the desired state.  Where possible, we identify and propose a fix, and submit a pull request to the [code owner](../code_reviews.md#who-should-i-get-a-code-review-from).
+
+**Medium changes**
+
+Medium changes operate on the same principle as small changes, with the exception of the team not submitting code changes. These changes also include a documented test plan for validation of the security change. 
+
+**Large changes**
```
Comment (nicksnyder) [inline handbook/engineering/security/index.md:106]: Where is the approximate line between medium and large? Medium is a few days, large is over a week?

File: handbook/engineering/security/index.md (hunk 4 above)
Comment (chayim) [reply handbook/engineering/security/index.md:116]: I wish - no one can ever pronounce the "CH".

File: handbook/engineering/security/index.md (hunk 3 above)
Comment (chayim) [reply handbook/engineering/security/index.md]: I like the active voice change you've highlighted in your writing. It feels like a higher degree of ownership - thank you for sharing that.

File: handbook/engineering/security/index.md (hunk 5 above)
Comment (chayim) [reply handbook/engineering/security/index.md:106]: That feels about right.  Large is also based on scope such as cross cutting changes. Updated language.

File: handbook/engineering/security/index.md (hunk 4 above)
Comment (tsenart) [reply handbook/engineering/security/index.md]: I DO!

**Pull request merged**